lambda_src/drivers/process_https.py

```diff
@@ -90,15 +92,18 @@ def process_row(
             req_headers['authorization'] = req_auth['authorization']
         elif 'headers' in req_auth:
             req_headers.update(req_auth['headers'])
-
+        elif 'body' in req_auth:
+            pre_data=req_auth['body']
```

Comment:
variables should always have space between them and operators. So change to:

```
pre_data = req_auth['body']
req_data = pre_data.encode('utf-8')
```

Reply:
Done